src/checkstyle/checks.xml

```diff
@@ -164,6 +164,7 @@
         <module name="WhitespaceAround">
             <property name="allowEmptyConstructors" value="true" />
             <property name="allowEmptyMethods" value="true" />
+            <property name="allowEmptyLambdas" value="true" />
```

Comment:
Does it mean that now I can either use: `() -> { }` or `() -> {}` and checkstyle will be happy in both cases?

I would prefer to enforce one of them. 

Can we do the same with empty methods (constructors), I mean can enforce one way of formatting them: 

```
Constructror() {}
```
vs
```
Constructor() 
{
}
```

Maybe simple regex could be enough that disallows a white space character between `{}`?

Reply:
Yea, it is possible. I refrained from doing so, because currently the codebase uniformly uses `{\n}` for empty catch clauses (with single 1 exception). I didn't want to change that unilaterally, especially that it wouldn't necessarily improve readability.

OTOH, my main motivation was to reduce problems for developers. Lambdas are generally used in nested context, so making them more readable helps everyone. It's not obvious to me that being strict on empty constructors/methods is as helpful (it increases "oh Travis said i'm bad" ratio....)

Reply:
I'd personally prefer to enforce `{}` everywhere, as it's more consistent and signals that the block as intentionally left empty. But we should tackle that separately (assuming there is agreement for try/catch or other non-method blocks).